leo_docking/leo_docking/states/reach_docking_pose.py

```diff
@@ -234,51 +206,51 @@ def execute(self, ud):
         """Main state method, executed automatically on state entered"""
         self.reset_state()
```

Comment:
Nitpick: I might rename the ud variable to user_data as you have done on some other places. The ud is used here and in couple other places.